tests/gtests/sw_testhelpers.h

```diff
@@ -18,14 +22,54 @@ static const double
 #undef missing
 #define missing(x)  ( EQ( fabs( (x) ), SW_MISSING ) || !std::isfinite( (x) ) )
 
-extern SW_ALL SW_All;
-extern SW_OUTPUT_POINTERS SW_OutputPtrs;
-extern LOG_INFO LogInfo;
-extern PATH_INFO PathInfo;
+
+class AllTest : public::testing::Test {

```

Comment:
test fixtures are the way to go! It also appears that the tests are running slightly faster?